src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-09-17-studio-73-21-1.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'September 17, 2018'
+description: .
+tree_item_index: 985
+review:
+  comment: ''
+  date: '2018-09-17'
+  status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v73-21'}}
+Introducing Git access for Studio Designer.
+{{! /multiexcerpt}}
+
+## Git Access for Studio Designer
+If your project benefits from branch management features, you can request Git access for Studio Designer to our support team. This feature allows you to edit your Studio Designer Web UI customization from your favorite IDE, commit and push your changes back to Nuxeo Studio. Feel free to have a look at the [Studio Designer Git access documentation]({{page page='nuxeo-studio-designer-git-access'}}) for further details.
+
+## Noteworthy Improvements and Bugfixes
+- Studio will not generate an `assignees` node variable anymore when adding new workflow nodes  (([NXS-4772](https://jira.nuxeo.com/browse/NXS-4772))).
+- Studio will warn you on unsaved changes when using shortcut buttons to configure layouts (([NXS-4772](https://jira.nuxeo.com/browse/NXS-4857))).
+- [Designer / Web UI] Removed a false positive warning that could appear when configuring actions (([NXS-4812](https://jira.nuxeo.com/browse/NXS-4812))).
+- [Designer / Web UI] Studio now offers to completely separate Studio Designer's automatic configuration from your custom code (([NXS-4758](https://jira.nuxeo.com/browse/NXS-4758))).
+- [Designer / Web UI] Latest contributions are available in Studio Designer (([NXS-4799](https://jira.nuxeo.com/browse/NXS-4799))).
+- [Designer / Web UI] Advanced options are now easier to display (([NXS-4756](https://jira.nuxeo.com/browse/NXS-4756))).
+- Default configuration is now to search on all entries when configuring a predicate on a multivalued complex field (([NXS-4847](https://jira.nuxeo.com/browse/NXS-4847))).
+- [Designer / Web UI] When configuring an aggregate using a dropdown list, generated code is now correct (([NXS-4350](https://jira.nuxeo.com/browse/NXS-4350))).
+- A direct link to a workflow graph doesn't generate an error anymore (([NXS-4806](https://jira.nuxeo.com/browse/NXS-4806))).
+- [Designer / Web UI] Correct code is now generated when configuring a justified grid for Nuxeo Server 10.2 (([NXS-4846](https://jira.nuxeo.com/browse/NXS-4846))).
+- [Designer / Web UI] "Sortable" option when configuring a table layout works again in all cases (([NXS-4846](https://jira.nuxeo.com/browse/NXS-4846))).
+- Studio will provide a validation error when it detects an invalid predicate configuration that could prevent Studio Designer from loading (([NXS-4819](https://jira.nuxeo.com/browse/NXS-4819))).
```

Comment:
- **Studio provides** a validation error when it detects an invalid predicate configuration that could prevent Studio Designer from loading (([NXS-4819](https://jira.nuxeo.com/browse/NXS-4819))).